### PR TITLE
paru: add page

### PR DESCRIPTION
**Pull request by silentnoodlemaster**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/paru.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# paru
+
+> An AUR helper and pacman wrapper.
+> More information: <https://github.com/morganamilo/paru>.
+
+- Interactvley seach and install a package:
```
Comment (owenvoke) [inline pages/linux/paru.md:6]:
```suggestion
- Interactively search for and install a package:
```

File: pages/linux/paru.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`paru {{package_name_or_seach_term}}`
+
+- Synchronize and update all packages:
+
+`paru`
+
+- Upgrade AUR pacckages:
```
Comment (owenvoke) [inline pages/linux/paru.md:14]:
```suggestion
- Upgrade AUR packages:
```

File: pages/linux/paru.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`paru -Sua`
+
+- Get information abaout a package:
```
Comment (owenvoke) [inline pages/linux/paru.md:18]:
```suggestion
- Get information about a package:
```

**Review by owenvoke — CHANGES_REQUESTED** [review]
Hi @silentnoodlemaster, thanks for the new page. Just a couple of minor typo changes below, but otherwise looks good to me. 👍🏻

**Review by owenvoke — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @silentnoodlemaster!

**Pull request merged**